### PR TITLE
ManifestDb HWP logic fix

### PR DESCRIPTION
**Pull request by kmharrington**
This fixes the logic for making and/or connecting to ManifestDbs with the HWP angles. Previously the behavior was creating a new database every time it was run. 

Also includes a fix to the Imprinter CLI to account better for an books we bind outside the docker image.

### CODE REVIEW

File: sotodlib/hwp/g3thwp.py [hunk 1]
```diff
@@ -535,7 +535,7 @@ def write_solution(self, solved, output=None):
         solved: dict
           dict data from analyze
         output: str or None
-          output path + file name, overwirte config file
+          output path + file name, overwrite config file
```
Comment (mhasself) [inline sotodlib/hwp/g3thwp.py:538]: override

File: sotodlib/io/imprinter_cli.py [hunk 2]
```diff
@@ -79,6 +82,13 @@ def check_failed_books(imprint:Imprinter):
             imprint.bind_book(
                 book, ignore_tags=ignore_tags, ancil_drop_duplicates=ancil_drop_duplicates
             )
+            if imprint.docker_output_root is not None:
```
Comment (mhasself) [inline sotodlib/io/imprinter_cli.py:85]: I feel like you need a comment somewhere explaining what's going on with this docker_output_root business.  The behavior here seems a bit more complex than "Overide imprinter output root?"  Maybe it's simpler than I'm thinking, which is I might deserve some reassuring comment.

**Review by mhasself — APPROVED** [review]
Looks fine ... couple requests for doc fixes but I'll understand if they're not a priority!

**Review by yukisakurai — APPROVED** [review]
Thank you, I confirmed it works normally for both satp1 and satp3. I had to delete the output files every time during debugging..., so this is very helpful!

**Pull request merged**